rmfuse/fuse.py

```diff
@@ -389,6 +390,17 @@ def parse_args():
     parser.add_argument('--version', action='version', version=VERSION)
     return parser.parse_args()
 
+def isconnected(host='1.1.1.1', port=80, timeout=1):
+    # timeout is expressed in seconds
+    try:
+        s = socket.create_connection((host, port), timeout)
+        if s is not None:
+            s.close()
```

Comment:
What are the conditions that would cause `s` to be None?  Should this be considered an error?  If so, moving the `return True` inside of this block would be the quick fix.